defs/scenarios/openstack/neutron/bugs.yaml

```diff
@@ -47,6 +52,15 @@ checks:
       path: 'var/log/neutron/neutron-l3-agent.log'
     expr: '.+Gateway interface for router \S+ was not set up; router will not work properly'
 conclusions:
+  lp1794991:
+    decision: has_1794991
+    raises:
+      type: LaunchpadBug
+      bug-id: 1794991
+      message: >-
+        neutron-openvswitch-agent is not restarted after restart of
```

Comment:
Suggested amendment to msg:

This host may be affected by a bug in Openstack Neutron ML2 whereby if the neutron-openvswitch-agent service is not restarted after a restart of openvswitch-switch this can lead to inconsistent l2pop flows. If you think this node is impacted you can fix this with a restart of neutron-openvswitch-agent.

Reply:
Modified the message and added unit test cases